server/container/guice/cassandra-rabbitmq-guice/src/main/java/org/apache/james/CassandraRabbitMQJamesServerMain.java

```diff
@@ -33,10 +34,13 @@
 import com.google.inject.util.Modules;
 
 public class CassandraRabbitMQJamesServerMain {
-    public static final Module MODULES =
-        Modules
-            .override(Modules.combine(REQUIRE_TASK_MANAGER_MODULE, new DistributedTaskManagerModule()))
-            .with(new RabbitMQModule(), new BlobStoreChoosingModule(), new RabbitMQEventBusModule(), new TaskSerializationModule());
+    public static final Module MODULES = Modules
+        .override(Modules.combine(REQUIRE_TASK_MANAGER_MODULE, new DistributedTaskManagerModule()))
+        .with(new RabbitMQModule(),
```

Comment:
not related to your work but why are we overriding here?

Reply:
Not related yes but good point... I guess you mean that all those modules could be just in a `Modules.combine` right?

Reply:
yes

Reply:
Many binding conflict issues in this case... I will revert as it's not the object of this PR anyway